plugins/normalize-whitespace/demo.md

```diff
@@ -0,0 +1,53 @@
+---
```

Comment:
As I said in the same v2 change, I think this should be part of the readme, possibly as a code snippet with a certain class that we pick up on the website and transform it to a live example.

Reply:
Yup, I remember that. Will do. We just need to start the process, so we have something to work with.